src/options/converge.ts

```diff
@@ -22,6 +22,7 @@ export function convergeOptions(options: ParserOptions & PugParserOptions): PugP
 		closingBracketPosition: options.pugClosingBracketPosition ?? options.closingBracketPosition,
 		commentPreserveSpaces: options.pugCommentPreserveSpaces ?? options.commentPreserveSpaces,
 		pugSortAttributesBeginning: options.pugSortAttributesBeginning,
-		pugSortAttributesEnd: options.pugSortAttributesEnd
+		pugSortAttributesEnd: options.pugSortAttributesEnd,
+		pugMaxAttributesOnFirsLine: options.pugMaxAttributesOnFirsLine
```

Comment:
You have a typo in your copy-paste 😛

Reply:
Oh hahaha. It was late!

Reply:
It was actually a sloppy search'n'replace 🤕